tests/h/search/query_test.py

```diff
@@ -513,11 +513,32 @@ def search(self, search):
 
 
 class TestUsersAggregation(object):
+    def test_it_aggregates_annotations_on_users(self, Annotation, search):
+        for i in range(2):
+            Annotation(userid="acct:pa@example.com")
+        Annotation(userid="acct:pb@example.com")
 
-    @pytest.fixture
-    def search(self, search):
         search.append_aggregation(query.UsersAggregation())
-        return search
+        result = search.run({})
+
+        count_for_pa = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pa@example.com")['count']
+        count_for_pb = next(r for r in result.aggregations.get('users') if r["user"] == "acct:pb@example.com")['count']
+
+        assert len(result.aggregations.get('users')) is 2
+        assert count_for_pa is 2
```

Comment:
This should be fine given the small number of documents, but a caveat with aggregations in general is that the document counts are approximate.

Reply:
👍 Yeah, I read that in the documentation. I think this small number is good for tests.